main.tf

```diff
@@ -66,7 +66,11 @@ module "sns_topic" {
   attributes      = concat(module.this.attributes, ["config"])
   subscribers     = var.subscribers
   sqs_dlq_enabled = false
-  tags            = module.this.tags
+
+  kms_master_key_id           = var.sns_encryption_key_id
+  sqs_queue_kms_master_key_id = var.sqs_queue_kms_master_key_id
```

Comment:
can we expose `encryption_enabled` too ?

Reply:
I was on the fence about upgrading the child module version, when contributing what should I stick to?

0.15.0 has no `encryption_enabled`

Reply:
Initially when you'd commented though, I was using `0.19.2` which had that option.